config/schedule.rb

```diff
@@ -0,0 +1,3 @@
+every 1.day, at: '4:30 am' do
+  rake 'feed:store' # from the PACs engine
```

Comment:
@osharpe is invoking `rake` enough or do we need to have a custom command to cd into the right directory and so on?

Reply:
@your you will need to cd and source environment variables to make this work. Something like `cd /srv/frontend/ && source /etc/mas/environment && RAILS_ENV=production bundle exec rake feed:store` We will also need to make some puppet changes to setup the whenever gem cron on the hosts as currently its only setup on CMS host in Prod, this is something I can help with in due course.

Reply:
@osharpe makes sense, I'll keep that in mind. I can amend at least the first part in the while, thanks :)